src/_pytest/python.py

```diff
@@ -942,6 +960,24 @@ def make_unique_test_ids(self) -> List[str]:
 
         return resolved_ids
 
+    def make_parameter_keys(self) -> Iterable[Dict[str, Hashable]]:
+        """Make hashable keys for parameters in a parametrized test.
+
```

Comment:
Will add a paragraph describing the return value:

For each parameter set, generates a dict mapping each parameter to its key.